src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Value/TimeOnlyConverter.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers.Text;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace System.Text.Json.Serialization.Converters
+{
+    internal sealed class TimeOnlyConverter : JsonConverter<TimeOnly>
```

Comment:
>TimeOnlyConverter [](http://example.com/codeflow?start=26&length=17)

The implementation here looks ok, but I am wondering why we didn't use the same idea we used with DateOnly? I mean use DateTime for serialization and deserialization? The difference is you'll not need to check the bounds `if (timespan < TimeSpan.Zero || timespan > s_timeOnlyMaxValue)`.